h2o-docs/src/product/data-science/algo-params/single_node_mode.rst

```diff
@@ -0,0 +1,42 @@
+``single_node_mode``
+--------------------
+
+- Available in: CoxPH, Deep Learning
+- Hyperparameter: yes
+
+Description
+~~~~~~~~~~~
+
+Specify whether to run on a single node for fine-tuning of model parameters. Running on a single node reduces the effect of network overhead for smaller datasets.
+
+Example
+~~~~~~~
+
+.. tabs::
+	.. code-tab:: r R
+
+		library(h2o)
+		h2o.init()
+
+		# Import the heart dataset:
+		heart <- h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/coxph_test/heart.csv")
+
+		# Build and train the model:
+		coxph.model <- h2o.coxph(x="age", event_column="event", start_column="start", stop_column="stop",training_frame=heart, single_node_mode=TRUE)
+
```

Comment:
Add a step to get the predictions:
pred <- h2o.predict(object=coxph.model, newdata=heart)